.gitignore

```
@@ -4,3 +4,4 @@
 .settings
 .vscode
 target
+myconvert.gpx
```

I assume this was a leftover from a local test, but you could replace it with *.gpx and *.tcx to avoid accidentally committing these files.
```suggestion
*.gpx
*.tcx
```